network/quic/sessionmanager.go

```diff
@@ -0,0 +1,100 @@
+package quic
+
+import (
+	"time"
+
+	h "github.com/ConsenSys/handel"
+	quic "github.com/lucas-clemente/quic-go"
+)
+
+// The handel protocol is very well suited to fire and forget type of
+// transport (udp for example), with quic or other type of statefull protocol
```

Comment:
`of stateless protocol` you meant no ?

Reply:
hmm quic, maintains session so I thin it is statefull..?